exercises/prime-factors/prime_factors_test.go

```diff
@@ -1,15 +1,16 @@
 package prime
 
-// Return prime factors in increasing order
-
 import (
 	"reflect"
+	"sort"
 	"testing"
 )
 
 func TestPrimeFactors(t *testing.T) {
 	for _, test := range tests {
 		actual := Factors(test.input)
+		sort.Slice(actual, ascending(actual))
+		sort.Slice(test.expected, ascending(test.expected))
```

Comment:
I think this needs to be:

```suggestion
		sort.Slice(test.expected, ascending)
```

Same above.

`func Slice(slice interface{}, less func(i, j int) bool)` expects a function as second argument.

Reply:
Yes, my function returns a closure.

Reply:
see also the example on https://golang.org/pkg/sort/#Slice

Reply:
right...  my bad. I guess I'm tired :D